### PR TITLE
serving users

### PR DESCRIPTION
**Pull request by jamil314**
[core pr](https://github.com/opencrvs/opencrvs-core/pull/5677)

### CODE REVIEW

File: src/features/employees/source/default-employee.csv
Comment (Zangetsu101) [inline src/features/employees/source/default-employee.csv:1]: Could you please change the mobile & email values to be unique for each row?

File: src/features/employees/source/default-employee.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
+primaryOfficeId,givenNames,familyName,role,type,mobile,email,password
```
Comment (Zangetsu101) [inline src/features/employees/source/default-employee.csv]:
```suggestion
primaryOfficeId,givenNames,familyName,sysntemRole,role,mobile,email,password
```

File: src/features/employees/validator.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      'Local Registrar',
+      'Registraire local',
+      'Local System Admin',
+      'Administrateur système local',
+      'National System Admin',
+      'Administrateur système national',
+      'Performance Manager',
+      'Gestion des performances',
+      'National Registrar',
+      'Registraire national'
```
Comment (Zangetsu101) [inline src/features/employees/validator.ts]: do we need the french labels here?

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: src/features/employees/source/default-employee.csv
Comment (jamil314) [reply src/features/employees/source/default-employee.csv:1]: Mobile was already unique, updated emails to be unique

File: src/features/employees/source/default-employee.csv (hunk 1 above)
Comment (jamil314) [reply src/features/employees/source/default-employee.csv]: updated

File: src/features/employees/validator.ts (hunk 2 above)
Comment (jamil314) [reply src/features/employees/validator.ts]: removed french labels as the are not used

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**